src/modules/notifications/notifications.service.ts

```diff
@@ -1,38 +1,82 @@
-import { Injectable, Logger } from "@nestjs/common";
-import { FirebaseService } from "../firebase/firebase.service";
-import { HandleException } from "src/common/exceptions/handler/handle.exception";
-import { UsersService } from "../users/users.service";
-import { NotificationDTO } from "../firebase/models/firebase.request.dto";
-import { title } from "process";
-import { UpdateAppRequestDTO } from "./models/update.app.request.dto";
-import { stringConstants } from "src/utils/string.constant";
+import { Injectable, Logger } from '@nestjs/common';
+import { FirebaseService } from '../firebase/firebase.service';
+import { HandleException } from 'src/common/exceptions/handler/handle.exception';
+import { UsersService } from '../users/users.service';
+import { NotificationDTO } from '../firebase/models/firebase.request.dto';
+import { title } from 'process';
+import { UpdateAppRequestDTO } from './models/update.app.request.dto';
+import { stringConstants } from 'src/utils/string.constant';
+import { SendCustomNotificationDTO } from './models/send.custom.notification.dto';
 
 @Injectable()
 export class NotificationsService {
   constructor(
     private readonly firebaseService: FirebaseService,
     private readonly userService: UsersService,
-  ) {} 
-Ò
-
+  ) {}
+  Ò;
```

Comment:
do you need this?

Reply:
No, It was a typing error. It is ok now.